src/test/java/guitests/PanelRenameTest.java

```diff
@@ -151,7 +152,34 @@ public void panelRenameTest() {
         Text panelNameText7 = find("#dummy/dummy_col7_nameText");
         assertEquals(expected, panelNameText7.getText());
         sleep(EVENT_DELAY);
-        
+
+        // Testing whether the close button appears once rename box is opened.
+        // Expected: Close button should not appear once rename box is opened and while edits are being made.
+        //           It should appear once the rename box is closed and the edits are done.
+        press(CREATE_RIGHT_PANEL);
+        boolean isPresentBeforeEdit = exists("#dummy/dummy_col8_closeButton");
+        PlatformEx.runAndWait(() -> UI.events.triggerEvent(new ShowRenamePanelEvent(8)));
+        sleep(EVENT_DELAY);
+        boolean isPresentDuringEdit = true; //stub value, this should change to false.
+        try {
+            exists("#dummy/dummy_col8_closeButton");
+        } catch (NoNodesFoundException e){
+            isPresentDuringEdit = false;
+        }
+
+        String randomName8 = RandomStringUtils.randomAlphanumeric(PANEL_MAX_NAME_LENGTH - 1);
+        PanelNameTextField renameTextField8 = find("#dummy/dummy_col8_renameTextField");
+        renameTextField8.setText(randomName8);
+        push(KeyCode.ENTER);
+        boolean isPresentAfterEdit = exists("#dummy/dummy_col8_closeButton");
+        Text panelNameText8 = find("#dummy/dummy_col8_nameText");
+        assertEquals(true, isPresentBeforeEdit);
+        assertEquals(false, isPresentDuringEdit);
+        assertEquals(true, isPresentAfterEdit);
+        assertEquals(randomName8, panelNameText8.getText());
+        sleep(EVENT_DELAY);
```

Comment:
Don't use sleep in new tests. Find some other way to synchronise. If there might be lag, you can use PlatformEx.waitOnFxThread(), otherwise wait for some condition to be true or some node to be visible.